state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -461,6 +467,12 @@ pub mod pallet {
 			deposit_details: <T::TargetChain as Chain>::DepositDetails,
 			reason: DepositIgnoredReason,
 		},
+		EgressIgnored {
+			egress_id: EgressId,
+			asset: TargetChainAsset<T, I>,
+			amount: TargetChainAmount<T, I>,
+			destination_address: TargetChainAccount<T, I>,
```

Comment:
should we add a `reason` enum? future-proof it?

Reply:
done